packages/sdk/src/conversationWithCloudAdapter/conversation.ts

```diff
@@ -188,12 +188,14 @@ export class ConversationBot {
    * For example, to use with Restify:
```

Comment:
This should be changed as well. restify -> express?

Reply:
Updated, thanks!